### ISSUE TITLE
No users found..

### ISSUE THREAD

**Issue by hardwarehackr** [aarmea/noise#16]
Hi,

I compiled your project and it looks very promising.. 

I have it installed on 2 phones and have the MAC address set in the settings, however there is no other users ever found?

I'm not sure if I missed something?

cheers

**Comment by ktorn** [comment]
@hardwarehackr Not sure what you mean by users. You mean in the UI?

I've just tried the `master` branch and under the `Debug: raw messages` screen I can see messages being communicated between two devices in close proximity. I think that's about it for the current (early) implementation.

**Comment by hardwarehackr** [comment]
Yes, sorry, within the UI there is no users or any way to send messages.. 

Ill revisit it and have a look in the debug. 

Cheers

**Comment by ktorn** [comment]
@hardwarehackr that's right, at least on `master` the main screen does nothing.

Check my modified [UnknownMessage toString method](https://github.com/ktorn/noise/blob/e3a793c37216da262f3634f188a2f1c33f3d6a0a/app/src/main/java/com/alternativeinfrastructures/noise/storage/UnknownMessage.kt) which makes debug messages slightly more human-readable. Then when running the app, click on the hamburger icon (top-right) and `Debug: raw messages`. You should see the initial IdentityAnnouncementMessages, followed by one or more UnknownMessages as you click on the pink message send icon.

**Comment by hardwarehackr** [comment]
Great.. ill check it out.. 

any chance you can email me? my username @pm.me or twitter?
